acid/acid.py

```diff
@@ -1,16 +1,19 @@
 """An XBlock checking container/block relationships for correctness."""
 
 import logging
-import pkg_resources
 import random
+
+import pkg_resources
 import webob
 from lazy import lazy
 from mako.lookup import TemplateLookup
-
 from xblock.core import XBlock, XBlockAside
-from xblock.fields import Scope, Dict
-from xblock.fragment import Fragment
-import six
+from xblock.fields import Dict, Scope
+
+try:
+    from xblock.fragment import Fragment
+except:
+    from web_fragments.fragment import Fragment
```

Comment:
We should put it in try 
Like we implemented [here](https://github.com/farhan/xblock-drag-and-drop-v2/blob/9754a3e658e4d0e335f3bbd7acf3e9d00259212f/drag_and_drop_v2/drag_and_drop_v2.py#L26)

Reply:
This is missing a comment that explains why we're doing this.  Please add something like https://github.com/farhan/xblock-drag-and-drop-v2/blob/9754a3e658e4d0e335f3bbd7acf3e9d00259212f/drag_and_drop_v2/drag_and_drop_v2.py#L29 so later we know why we did this without having to dig into git history.